app/javascript/bundles/Container/components/lessons/AlkaliMetals.js

```diff
@@ -0,0 +1,183 @@
+            controls.rotateSpeed = 0.5;
+            controls.minDistance = 500;
+            controls.maxDistance = 6000;
+            controls.addEventListener( 'change', render );
+            transform( targets.sphere, 2000 );
+            var button = document.getElementById( 'table' );
+            button.addEventListener( 'click', function () {
+                if(temp.length > 0) {
+                    objects = temp;
+                    temp = [];
+                }
+                transform( targets.table, 2000 );
+            }, false );
+    
+            var button = document.getElementById( 'helix' );
+            button.addEventListener( 'click', function () {
+                temp = objects;
+                objects = alkali;
+                transform( targets.helix, 2000 );
+            }, false );
+            //
+            window.addEventListener( 'resize', onWindowResize, false );
+        }
+
+        function transform( targets, duration ) {
+            TWEEN.removeAll();
+            for ( let i = 0; i < objects.length; i ++ ) {
+                var object = objects[ i ];
+                let target = targets[ i ];
+                new TWEEN.Tween( object.position )
+                    .to( { x: target.position.x, y: target.position.y, z: target.position.z }, Math.random() * duration + duration )
+                    .easing( TWEEN.Easing.Exponential.InOut )
+                    .start();
+                new TWEEN.Tween( object.rotation )
+                .to( { x: target.rotation.x, y: target.rotation.y, z: target.rotation.z }, Math.random() * duration + duration )
+                .easing( TWEEN.Easing.Exponential.InOut )
+                .start();
+            }
+            new TWEEN.Tween( this )
+                .to( {}, duration * 2 )
+                .onUpdate( render )
+                .start();
+        }
+
+        function onWindowResize() {
+            camera.aspect = window.innerWidth / window.innerHeight;
+            camera.updateProjectionMatrix();
+            renderer.setSize( window.innerWidth, window.innerHeight );
+            render();
+        }
+
+        function animate() {
+            requestAnimationFrame( animate );
+            TWEEN.update();
+            controls.update();
+        }
+
+        function render() {
+            renderer.render( scene, camera );
+        }
+    
+        let prevScrollpos = window.pageYOffset;
+        window.onscroll = function() {
+            let currentScrollPos = window.pageYOffset;
+            if (prevScrollpos > currentScrollPos) {
+                document.getElementById("to-ammend").style.top = "45px";
+            } else {
+                document.getElementById("to-ammend").style.top = "-50px";
+            }
+            prevScrollpos = currentScrollPos;
+        }
+    })
+       
+    return (
+        <div id="alkalin">
+           <section id="top" className="lesson-container">
+                <div className="main">
```

Comment:
i'd remove this `id="top"` since it adds a fixed height on the top div of the lesson which is not so nice I think